leval/evaluator.py

```diff
@@ -7,14 +8,23 @@
     InvalidConstant,
     InvalidNode,
     InvalidOperation,
+    NoSuchValue,
     Timeout,
     TooComplex,
 )
 from leval.universe.base import BaseEvaluationUniverse
 from leval.utils import expand_name
 
+if sys.version_info < (3, 10):
+    NoneType = type(None)
+else:
+    from types import NoneType
```

Comment:
You shouldn't need to import `sys` and do a version comparison; doesn't just

```suggestion
try:
    from types import NoneType
except ImportError:
    NoneType = type(None)
```
do the trick?